AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/MainActivity.java

```diff
@@ -152,6 +153,7 @@ private Drawable getMuteIconForCurrentSdkMuteSetting()
         items.add( new AdType( "Rewarded", new Intent( this, RewardedAdActivity.class ) ) );
         items.add( new AdType( "Banners", new Intent( this, BannerAdActivity.class ) ) );
         items.add( new AdType( "MRECs", new Intent( this, MrecAdActivity.class ) ) );
+        items.add( new MediationDebuggerType( "Launch Mediation Debugger" ) );
```

Comment:
Maybe we can refactor `AdType` to be a more generic class, and have the ability to take in an Intent to launch _or_ a `Runnable` block to execute? Otherwise in the future when we add more option items, I don't want to create a new class for each.